src/commands/misc/listing.md

```diff
@@ -17,7 +17,7 @@ bdbd3439  2015-05-08 21:45:17  luigi          /home/art
 You can filter the listing by directory path:
 
 ```console
-$ rustic -r /srv/rustic-repo snapshots --path="/srv"
+$ rustic snapshots --filter-paths="/srv"
```

Comment:
Wasn't it singular in CLI and plural in the config?

Reply:
Not for filter-paths. Here it is not meant that we may specify multiple paths fields, but that each field may contain multiple values. For instance: `--filter-paths="/home,/root"`